contrib/inventory/gce_googleapiclient.py

```diff
@@ -189,7 +206,7 @@ def main(args):
         ARGS = docoptcfg(DOCOPT_USAGE, env_prefix=ENV_PREFIX)
 
     log.debug(ARGS)
-    if None in ARGS.values():
+    if not ARGS['--account']:
```

Comment:
we only need `--account` if we are getting all projects, so don't make it as required. also, if you want to enforce a parameter, you'd have to do it using docopts, not manually checking in the code.